### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by salaboy**
Adding links to other sections and refactor Continuous Delivery to Core

### CODE REVIEW

**Review by erkist — APPROVED** [review]
Looks good!

**Review by m-linner-ericsson — APPROVED** [review]

File: vocabulary-draft/introduction.md [hunk 1]
```diff
@@ -8,10 +8,11 @@ These events are agnostic from any specific tool and are designed to fit a wide
 
 The phases covered by this proposal are:
 
-- **Source Code Version Control Events**: Events emitted by changes in source code or by the creation, modification or deletion of new repositories that hold source code.
-- **Continuous Integration Pipelines Events**: includes events related to building, testings, packaging and releasing software artifacts, usually binaries.
-- **Continuous Deployment Pipelines Events**: include events related with environments where the artifacts produced by the integration pipelines actually run. These are usually services running in a specific environment (dev, QA, production), or embedded software running in a specific hardware. 
-- **Continuous Delivery Events**: includes events related to activities and orchestration that needs to exist to be able to deterministically and continously being able to delivery software to users. 
+- **[Core Events](https://github.com/cdfoundation/sig-events/blob/main/vocabulary-draft/core.md)**: includes core events related to core activities and orchestration that needs to exist to be able to deterministically and continously being able to delivery software to users.
```
Comment (afrittoli) [inline vocabulary-draft/introduction.md:11]: Thank you! Could you replace the links with relative ones?
Absolute links are not very portable across branches or published docs...

Comment (salaboy) [reply vocabulary-draft/introduction.md]: done!

**Comment by salaboy** [comment]
@afrittoli I think this is ready to be merged

**Comment by salaboy** [comment]
@afrittoli I think this is ready to be merged..

**Review by afrittoli — APPROVED** [review]
Nice, thank you!

**Pull request merged**